ansible_collections/arista/avd/molecule/eos_designs_unit_tests/custom_modules/custom_interface_descriptions.py

```diff
@@ -0,0 +1,85 @@
+from functools import cached_property
+
+from ansible_collections.arista.avd.plugins.plugin_utils.utils import get
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.interface_descriptions import AvdInterfaceDescriptions as BaseAvdInterfaceDescriptions
+
+
+class AvdInterfaceDescriptions(BaseAvdInterfaceDescriptions):
```

Comment:
I would rather write it like
```suggestion
class CustomAvdInterfaceDescriptions(AvdInterfaceDescriptions):
```

so that it gives a simple example for people on how we want it to be used

Reply:
fixed